site2/website/pulsar-manager-release-notes.md

```diff
@@ -1,19 +1,39 @@
 ## Apache Pulsar Manager
 
+### 0.2.0 &mdash; 2020-09-28 <a id="0.2.0"></a>
+
+#### Fixes
```

Comment:
I suggest that we just remove this heading.
For guides, refer to [heading usage](https://developers.google.com/style/headings)